### DISCUSSION TITLE
Practicas FrontEnd

### DISCUSSION THREAD

**Discussion by May-Alvarez** [Launch-X-Latam/MisionFrontEnd#329] [Q&A] [answered]
Hola, por cuetiones de salud no eh podido hacer ninguna practica de la mision, tengo entendido que hoy se entrega el Abogabot y HTML, saben si se puede entregar despues de la fecha?

**Comment by Standard-IO** [comment, marked as answer]
No hay fechas obligatorias para entregar las "tareas", puedes hacerlas a tu ritmo. Por decirlo de alguna manera la fecha es tentativa y es para que no se acumule el trabajo. El propósito mayor de hacerlas y subirlas a tu repositorio es que puedas comprobar que tienes los conocimientos y  en un futuro cuando postules a un trabajo de desarrollador tengas como comprobarlo. Saludos y mejorarte.
